src/QMCHamiltonians/NonLocalECPComponent.h

```diff
@@ -77,7 +77,7 @@ class NonLocalECPComponent : public QMCTraits
   std::vector<ValueType> wvec;
 
   //Position delta for virtual moves.
-  std::vector<PosType> deltaV;
+  std::vector<PosType> deltaV_;
```

Comment:
Looks like you're updating the private data members that are being touched in this PR to have the trailing _
It is probably worth updating this class to follow the formatting standards for all its private data members, but that should be a subsequent PR. 

Other class like SOECPComponent already do this, so it would nice to have this one updated as well to help with readability